.prospector.yaml

```diff
@@ -0,0 +1,17 @@
+test-warnings: true
```

Comment:
can we, in general, comment the configs in this file when necessary ?

Reply:
Yes we can add comments if we want